### PR TITLE
fix: typo

### PR DESCRIPTION
**Pull request by Bojidar00**
Closes: #700

### CODE REVIEW

**Review by miguelmtzinf — APPROVED** [review]

**Comment by miguelmtzinf** [comment]
Thanks for the fix @Bojidar00 !

**Comment by The-3D** [comment]
can we point this to the 3.0.1 branch?

File: contracts/protocol/pool/DefaultReserveInterestRateStrategy.sol [hunk 1]
```diff
@@ -191,8 +191,8 @@ contract DefaultReserveInterestRateStrategy is IReserveInterestRateStrategy {
   }
 
   /// @inheritdoc IReserveInterestRateStrategy
-  function calculateInterestRates(DataTypes.CalculateInterestRatesParams calldata params)
-    external
+  function calculateInterestRates(DataTypes.CalculateInterestRatesParams memory params)
+    public
```
Comment (kyzia551) [inline contracts/protocol/pool/DefaultReserveInterestRateStrategy.sol:195]: @miguelmtzinf is it needed?

Comment (miguelmtzinf) [reply contracts/protocol/pool/DefaultReserveInterestRateStrategy.sol:195]: These changes are already part of the code (github is not calculating the diff correctly). They have been introduced per BGD's request (for the V2-V3 migration). 
Here PRs: https://github.com/aave/aave-v3-core/pull/721 and https://github.com/aave/aave-v3-core/pull/712

**Pull request merged**